src/containers/Application.js

```diff
@@ -93,10 +93,19 @@ class Application extends Component {
 
 		// Set API URL, if not configured
 		if (this.Config.get('API_URL') == undefined) {
-			this.Config.addDefaults({
-				API_URL: window.location.protocol + '//' + window.location.host + '/api/',
-			});
+			const publicUrl = __CONFIG__["publicUrl"];
+			if (publicUrl != ""){
+				this.Config.addDefaults({
+					API_URL: publicUrl + '/api/',
+				});
+			}
+			else {
+				this.Config.addDefaults({
+					API_URL: window.location.protocol + '//' + window.location.host + '/api/',
```

Comment:
Well, I expected that this will go into basepath, not here.

Reply:
The logic of "API_URL" should be:
1) It is specified in the config, so it will be used
2) when it is not specified, the base path is taken and "/api" is added (no `/` at the end).

"publicUrl" will go away eventually, as discussed, don't reintroduce it.